client/components/PreviewNav.jsx

```diff
@@ -13,7 +13,7 @@ const PreviewNav = ({ owner, project }) => (
         <InlineSVG src={logoUrl} alt="p5.js logo" />
       </div>
       <Link className="nav__item" to={`/${owner.username}/sketches/${project.id}`}>{project.name}</Link>
-      <p className="toolbar__project-owner">by</p>
+      <p className="toolbar__project-owner">&nbsp; by &nbsp;</p>
```

Comment:
even though there are definitely other instances of it in the codebase, i think that using `&nbsp;` isn't best practice. i actually think that this whole chunk of html should be different, as i think it's slightly more standard:
```jsx
<p className="toolbar__project-name">
  <Link className="nav__item" to={`/${owner.username}/sketches/${project.id}`}>{project.name} 
  </Link>
  <span className="toolbar__project-owner">{" by "}</span>
  <Link className="nav__item" to={`/${owner.username}/sketches/`}>{owner.username}</Link>
</p>
```
the CSS would need to be shifted a bit too (`.toolbar__project-owner` changed and `.toolbar__project-name` added)